markets/retrievaladapter/client.go

```diff
@@ -72,7 +72,7 @@ func (rcn *retrievalClientNode) GetChainHead(ctx context.Context) (shared.TipSet
 // WaitForPaymentChannelAddFunds waits messageCID to appear on chain. If it doesn't appear within
 // defaultMsgWaitTimeout it returns error
 func (rcn *retrievalClientNode) WaitForPaymentChannelAddFunds(messageCID cid.Cid) error {
-	_, mr, err := rcn.chainapi.StateManager.WaitForMessage(context.TODO(), messageCID)
+	_, mr, err := rcn.chainapi.StateManager.WaitForMessage(context.TODO(), messageCID, 5, 10)
```

Comment:
I'd put these numbers in constants in `build/params_shared.go`

Reply:
Confidence seems reasonable looking at chains/forks in chainwatch, but I'd set the default timeout for something quite high like 72 (30min with 25 block time)

Reply:
Really we don't need the timeout at all as callers can just cancel the context